model/metrics/system-metrics.yaml

```
@@ -340,3 +340,26 @@ groups:
     instrument: updowncounter
     unit: "By"
     attributes: []
+
+  - id: metric.system.linux.memory.slab.usage
+    type: metric
+    metric_name: system.linux.memory.slab.usage
+    stability: experimental
+    brief: "Reports the memory used by the Linux kernel for managing caches of frequently used objects by state."
+    note: |
+      The sum over all `system.linux.memory.slab.state` values SHOULD equal the total slab memory available on the system, that is `system.linux.memory.slab.limit`.
+    instrument: updowncounter
+    unit: "By"
+    attributes:
+      - ref: system.linux.memory.slab.state
+
+  - id: metric.system.linux.memory.slab.limit
+    type: metric
+    metric_name: system.linux.memory.slab.limit
+    stability: experimental
+    brief: "Total slab memory used in the system."
+    note: |
+      Its value SHOULD equal the sum of `system.linux.memory.slab.state` over all states.
```

I see we have the same wording for the `system.memory.limit` but something bugs me here, since we don't sum the `state` but the `usage` 🤔 .

Shouldn't that be
```suggestion
      Its value SHOULD equal the sum of `system.linux.memory.slab.usage` over all states.
```
?

---

Good catch! Docs regenerated in https://github.com/open-telemetry/semantic-conventions/pull/1078/commits/b03736c7b84ea3243f8ebe22f758e426c57c3181